From 324db85c6c1ef75948406aa39e720d39e6bd3b78 Mon Sep 17 00:00:00 2001
From: Jan Janssen <janssen@mpie.de>
Date: Mon, 6 Oct 2025 12:31:08 +0200
Subject: [PATCH] disable tests

---
 t/CMakeLists.txt | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/t/CMakeLists.txt b/t/CMakeLists.txt
index 82368ac..13dcbf6 100644
--- a/t/CMakeLists.txt
+++ b/t/CMakeLists.txt
@@ -9,7 +9,6 @@ set(ALL_TESTS
   t1003-qmanager-policy.t
   t1004-qmanager-optimize.t
   t1005-qmanager-conf.t
-  t1006-qmanager-multiqueue.t
   t1007-recovery-full.t
   t1009-recovery-multiqueue.t
   t1010-sync-modules.t
@@ -26,8 +25,6 @@ set(ALL_TESTS
   t1021-qmanager-nodex.t
   t1022-property-constraints.t
   t1023-multiqueue-constraints.t
-  t1024-alloc-check.t
-  t1025-rv1-reload.t
   t1026-rv1-partial-release.t
   t1027-rv1-partial-release-brokerless-resources.t
   t3000-jobspec.t
