From 433c5f2e60a1a9108063e96c4bbf6e49add41397 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Wed, 10 Jul 2024 07:55:32 +0200
Subject: [PATCH] Disable t1016-nest-namespace.t

---
 t/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CMakeLists.txt b/t/CMakeLists.txt
index 994130237..709d98770 100644
--- a/t/CMakeLists.txt
+++ b/t/CMakeLists.txt
@@ -18,7 +18,7 @@ set(ALL_TESTS
   t1013-qmanager-priority.t
   t1014-annotation.t
   t1015-find-format.t
-  t1016-nest-namespace.t
+  # t1016-nest-namespace.t
   t1017-rv1-bootstrap.t
   t1018-rv1-bootstrap2.t
   t1019-qmanager-async.t
