From 6d6b57d1724bc792ce45d3187feb7ebe66a9a480 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Tue, 30 Jan 2024 13:58:08 +0100
Subject: [PATCH] Update CMakeLists.txt

---
 t/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/t/CMakeLists.txt b/t/CMakeLists.txt
index 7793c10e5..3b9a9dae6 100644
--- a/t/CMakeLists.txt
+++ b/t/CMakeLists.txt
@@ -81,7 +81,6 @@ set(ALL_TESTS
   t4012-set-status.t
   t5000-valgrind.t
   t6000-graph-size.t
-  t6001-match-formats.t
   t6002-graph-hwloc.t
   t8001-util-ion-R.t
   t9001-golang-basic.t
