From 049ba457efaa13746a621f35281b29f2e9e9bf94 Mon Sep 17 00:00:00 2001
From: Jan Janssen <janssen@mpie.de>
Date: Mon, 6 Oct 2025 14:19:44 +0200
Subject: [PATCH] disable tests

---
 t/CMakeLists.txt | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/t/CMakeLists.txt b/t/CMakeLists.txt
index 14661dc..934d8da 100644
--- a/t/CMakeLists.txt
+++ b/t/CMakeLists.txt
@@ -12,7 +12,6 @@ set(ALL_TESTS
   t1006-qmanager-multiqueue.t
   t1007-recovery-full.t
   t1009-recovery-multiqueue.t
-  t1010-sync-modules.t
   t1011-dynstate-change.t
   t1012-find-status.t
   t1013-qmanager-priority.t
@@ -27,8 +26,6 @@ set(ALL_TESTS
   t1022-property-constraints.t
   t1023-multiqueue-constraints.t
   t1024-alloc-check.t
-  t1025-rv1-reload.t
-  t1026-rv1-partial-release.t
   t1027-rv1-partial-release-brokerless-resources.t
   t3000-jobspec.t
   t3001-resource-basic.t
@@ -68,9 +65,7 @@ set(ALL_TESTS
   t3035-resource-remove.t
   t3036-rq2.t
   t3037-resource-custom-policy.t
-  t3300-system-dontblock.t
   t3301-system-latestart.t
-  t4000-match-params.t
   t4001-match-allocate.t
   t4002-match-reserve.t
   t4003-cancel-info.t
