From 9d952c1ab82f0773c86a1379ede12c4df7d07e28 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Tue, 30 Jan 2024 13:41:54 +0100
Subject: [PATCH] Update Makefile.am

---
 t/Makefile.am | 1 -
 1 file changed, 1 deletion(-)

diff --git a/t/Makefile.am b/t/Makefile.am
index af1c3c88e..e13b6b3b5 100644
--- a/t/Makefile.am
+++ b/t/Makefile.am
@@ -99,7 +99,6 @@ TESTS = \
     t5000-valgrind.t \
     t5100-issues-test-driver.t \
     t6000-graph-size.t \
-    t6001-match-formats.t \
     t6002-graph-hwloc.t \
     t8001-util-ion-R.t \
     t9001-golang-basic.t \
