From 3581107a8fe91185d48ee4b360429237ba309ef8 Mon Sep 17 00:00:00 2001
From: Jan Janssen <janssen@mpie.de>
Date: Mon, 6 Oct 2025 11:00:51 +0200
Subject: [PATCH] disable test

---
 t/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/t/CMakeLists.txt b/t/CMakeLists.txt
index bde68da..7b80f46 100644
--- a/t/CMakeLists.txt
+++ b/t/CMakeLists.txt
@@ -27,7 +27,6 @@ set(ALL_TESTS
   t1022-property-constraints.t
   t1023-multiqueue-constraints.t
   t1024-alloc-check.t
-  t1025-rv1-reload.t
   t1026-rv1-partial-release.t
   t1027-rv1-partial-release-brokerless-resources.t
   t2317-resource-shrink.t
